### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio. Low-contrast text is difficult or impossible for many users to read. Learn more. Failing Elements h2

### ISSUE THREAD

**Issue by lauravikanis** [lauravikanis/portfolio2021#16]
Low-contrast text is difficult or impossible for many users to read. 
(Failing Elements h2) White on Red is not readable enough (Sectionheadline "Contact"). 
--> Make it black on red to fix
